### PR TITLE
My code review suggestions.

### PR DESCRIPTION
**Pull request by lizbissinger**
Analysis of the program: 
The program has some beginner UI set up showing what there intention for the app is. The app still seems to require more backend configuring, but overall the idea is good. 

Was the program available in GitHub in time: 
Yes

Is the program documented/commented well: 
Although the program lacks comments I don't think they are  necessary yet due to lack of code. 

Does the program compile:
Yes

Rationale behind my changes:
The changes I made had the intention to clean up and clarify the code.

Links to three specific commits I made to my groups repository:
https://github.com/comstogm/ProjectIT3048C/commit/f4993440a831312646bf00ba58140f56b7af6f91
https://github.com/comstogm/ProjectIT3048C/commit/66f219d6fda5e2bfb5748f95604a4bc6d82e0b22
https://github.com/comstogm/ProjectIT3048C/commit/f32977a77ac2c5149c8b5289867ef5badebd1f44

Specific technical concepts I learned from this code review:
- I learned what "const" is and when to use it
- I learned what qualifies as a redundant quantifier name
- I learned to double check my chosen names for things such as functions

### CODE REVIEW

File: app/src/main/java/com/taskruler/MainActivity.kt [hunk 1]
```diff
@@ -32,13 +32,13 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    LogActivity("Android")
```
Comment (lizbissinger) [inline app/src/main/java/com/taskruler/MainActivity.kt:35]: “Greeting” is not longer the best description of the function creating the UI for the home page/initial page you see when launching the app. If not my suggestion (LogActivity) maybe Home, LogScreen, etc.

File: app/src/main/java/com/taskruler/RetrofitClientInstance.kt [hunk 2]
```diff
@@ -5,14 +5,14 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://wrytheruc.github.io/"
+    private const val BASE_URL = "https://wrytheruc.github.io/"
```
Comment (lizbissinger) [inline app/src/main/java/com/taskruler/RetrofitClientInstance.kt:8]: This variable is known at compile-time
https://codersee.com/retrofit-with-kotlin-the-ultimate-guide/

https://www.baeldung.com/kotlin/const-var-and-val-keywords

File: app/src/main/java/com/taskruler/RetrofitClientInstance.kt [hunk 3]
```diff
@@ -5,14 +5,14 @@ import retrofit2.converter.gson.GsonConverterFactory
...
 
     val retrofitInstance : Retrofit?
         get() {
             // has this object been created yet?
             if (retrofit == null) {
                 // create it
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```
Comment (lizbissinger) [inline app/src/main/java/com/taskruler/RetrofitClientInstance.kt:15]: Redundant quantifier name. A fully qualified name may not be necessary due to a static import. 
https://tech.okcupid.com/cutting-through-the-static-find-and-fix-issues-the-smart-way-with-android-studio-static-code-fb0bc99d9788

File: app/src/main/java/com/taskruler/MainActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/taskruler/MainActivity.kt:35]: Agreed.  Make function names self-describing.

**Comment by discospiff** [comment]
Good suggestions here.  I recommend merging this branch.

**Comment by wrytherUC** [comment]
Aaron is implementing the changes in Spinner_Branch branch.

**Pull request closed without merging**